docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,174 @@
+| *tableName* | table name to be scanned | String | Yes | N/A |
+| *columnsExpression* | Comma separated list of columns to select from the given table. | String | Yes | N/A |
+| *key* | Primary key column name | String | Yes | N/A |
+| *partitionCount* | Static partitions count | int | No | 1 |
+| *whereCondition* | Where condition for jdbc query | String | No | N/A |
+| *fetchSize* | Hint limiting the number of rows to fetch in a single call | int | No | 20000 |
+| *pollInterval* | Interval in milliseconds to poll the database table | int | No | 10000 |
+| *queueCapacity* | Capacity of queue which holds DB data before emiting | int | No | 4096 |
+| *batchSize* | Maximum number of tuples to emit in a single call to the emitTuples() callback (see explanation above). | int | No | 2000 |
+
+#### <a name="JDBCStore"></a>Properties of JDBC Store (BackendStore)
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *databaseDriver* |JDBC Driver class for connection to JDBC Store. This driver should be there in class path | String | Yes | N/A |
+| *databaseUrl* | Database url of the form jdbc:subprotocol:subname | String | Yes | N/A |
+| *connectionProps* | Comma separated connection properties e.g. user:xyz,password:ijk | String | Yes | N/A |
+
+Of these only `store`, `tableName`, `columnsExpression` and `key` are mandatory. Those properties can be set like this:
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.prop.tableName</name>
+  <value>mytable</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.columnsExpression</name>
+  <value>column1,column2,column4</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.key</name>
+  <value>keycolumn</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseDriver</name>
+  <value>com.mysql.jdbc.Driver</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseUrl</name>
+  <value>jdbc:mysql://localhost:3306/mydb</value>
+</property>
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.connectionProps</name>
+  <value>user:myuser,password:mypassword</value>
+</property>
+```
+
+* If you need to filter the table records, set `whereCondition` which will be added to the generated SQL query.
+* If you have table with a very large number or rows to scan set `partitionsCount` to a higher number to increase read parallelism.
+* The operator uses PreparedStatement, a precompiled SQL statement to fetch records from database table. You can set `fetchSize` as a hint to the database driver to restrict number of rows to fetch in one call. The remaining rows of resultset will be fetched in subsequent calls. Please note, some of the databases may not honor this hint. Please refer to database documentation to check recommended value for your database.
```

Comment:
"databases may not" -- please verify if this is a limitation of the DB or the _driver_; from some docs, I got the impression that some drivers may not support it and other drivers _for the same DB_ may.